### DISCUSSION TITLE
AWS SNS support

### DISCUSSION THREAD

**Discussion by ABAG603** [JasperFx/wolverine#1093] [General]
Hi,

It is written in the docs that SNS is not supported yet but will come shortly. Are there any timelines when to expect it?

Thanks

**Comment by jeremydmiller** [comment]
@ABAG603 Sorry for being so slow to answer, I didn't realize that we'd enabled GitHub discussions for Wolverine and most conversations happen on Discord. 

There are no concrete plans at the moment. It's obviously something that would happen with more interest or a company sponsoring the work. Honestly, nobody has brought this up recently
